docs/architecture.rst

```diff
@@ -36,31 +54,59 @@ These boxes contains the following components from the latest CentOS Core:
     * nfs-utils
     * PyYAML
     * libyaml-devel
-  * atomic - The `Atomic CLI <https://github.com/projectatomic/atomic>`_
+  * atomic - The `Atomic CLI`_
   * docker-registry
   * docker
-    
-    The docker daemon is configured to expose a TLS protected TCP port so that Vagrant host based tools can access it.  This is accomplished in three steps:
 
-    1. The systemd startup for docker inside of the ADB will generate TLS certificates if they don't already exist.
+    The docker daemon is configured to expose a TLS protected TCP port so that
+    Vagrant host based tools can access it.  This is accomplished in three
+    steps:
+
+    1. The systemd startup for docker inside of the ADB will generate TLS
+       certificates if they don't already exist.
+
     2. The user of the ADB exposes the docker daemon port in their Vagrantfile
-    3. The certificates are copied out by the `vagrant-adbinfo plugin <https://github.com/bexelbie/vagrant-adbinfo>`_.
+
+    3. The certificates are copied out by the `vagrant-adbinfo plugin`_
 
   * kubernetes
   * File Synchronization
 
     * Possibly via NFS for libvirt
     * Possibly via containerized extensions for VirtualBox
-    * Designed to make sure code can move from the Vagrant host to the ADB easily
+    * Designed to make sure code can move from the Vagrant host to the ADB
+      easily
+
+.. _Atomic CLI: https://github.com/projectatomic/atomic
+.. _vagrant-adbinfo plugin: https://github.com/bexelbie/vagrant-adbinfo
 
 Containers providing the following functionality:
-  * OpenShift - A PAAS environment for source-to-image container execution in a devops pattern
-  * `Atomic App <https://github.com/projectatomic/atomicapp>`_ - The `Nulecule <https://github.com/projectatomic/nulecule>`_ container application specification reference implementation
-  * `atomic-reactor <https://github.com/projectatomic/atomic-reactor>`_ - a python based container builder
-  * `DevAssistant <http://www.devassistant.org/>`_ - used to provide templates for `Dockerfiles <https://github.com/devassistant/dap-docker>`_ and `Nulecule Specification Files <https://github.com/devassistant/dap-nulecule>`_ - possibly using this container: https://github.com/devassistant/da-docker-nulecule - see `Issue #39 <https://github.com/projectatomic/adb-atomic-developer-bundle/issues/39>`_
+
+  * OpenShift - A PAAS environment for source-to-image container execution in a
+    devops pattern
+
+  * `Atomic App`_ - The `Nulecule`_ container application specification
+    reference implementation
+
+  * `atomic-reactor`_ - a python based container builder
+
+  * `DevAssistant`_ - used to provide templates for `Dockerfiles`_ and `Nulecule
```

Comment:
Unrelated to your edits directly, we need to move DA into the "future" as that is not yet ready or containerized, afaik.